assets/src/legacy/map.js

```diff
@@ -6017,7 +5995,8 @@ window.lizMap = function() {
       });
```

Comment:
I think you have to update the upper code. I think you have to check if the response is OK (status in the range 200–299):
```js
      // Get config
      const configRequest = fetch(OpenLayers.Util.urlAppend(lizUrls.config, OpenLayers.Util.getParameterString(lizUrls.params))).then(function (response) {
        if (!response.ok) {
            throw 'Config not loaded: '+response.status+' '+response.statusText
       }
        return response.json()
      });
      // Get key/value config
      const keyValueConfigRequest = fetch(OpenLayers.Util.urlAppend(lizUrls.keyValueConfig, OpenLayers.Util.getParameterString(lizUrls.params))).then(function (response) {
        if (!response.ok) {
            throw 'Key/value config not loaded: '+response.status+' '+response.statusText
       }
        return response.json()
      });
      // Get WMS, WMTS, WFS capabilities
      const WMSRequest = fetch(OpenLayers.Util.urlAppend(service, OpenLayers.Util.getParameterString({ SERVICE: 'WMS', REQUEST: 'GetCapabilities', VERSION: '1.3.0' }))).then(function (response) {
        if (!response.ok) {
            throw 'WMS GetCapabilities not loaded: '+response.status+' '+response.statusText
       }
        return response.text()
      });
      const WMTSRequest = fetch(OpenLayers.Util.urlAppend(service, OpenLayers.Util.getParameterString({ SERVICE: 'WMTS', REQUEST: 'GetCapabilities', VERSION: '1.0.0' }))).then(function (response) {
        if (!response.ok) {
            throw 'WMTS GetCapabilities not loaded: '+response.status+' '+response.statusText
       }
        return response.text()
      });
      const WFSRequest = fetch(OpenLayers.Util.urlAppend(service, OpenLayers.Util.getParameterString({ SERVICE: 'WFS', REQUEST: 'GetCapabilities', VERSION: '1.0.0' }))).then(function (response) {
        if (!response.ok) {
            throw 'WFS GetCapabilities not loaded: '+response.status+' '+response.statusText
       }
        return response.text()
      });
```
Docs https://developer.mozilla.org/en-US/docs/Web/API/Response